src/pods/generate/GenerateUtil.ts

```diff
@@ -16,8 +19,15 @@ export async function addGeneratedResources(identifier: ResourceIdentifier, sett
   const resources = generator.generate(identifier, settings);
   let count = 0;
   for await (const { identifier: resourceId, representation } of resources) {
-    await store.setRepresentation(resourceId, representation);
-    count += 1;
+    try {
+      await store.setRepresentation(resourceId, representation);
+      count += 1;
+    } catch (err: unknown) {
+      if (!(ConflictHttpError.isInstance(err) && err.message === 'Existing containers cannot be updated via PUT.')) {
+        const logger = getLoggerFor('addGeneratedResources');
+        logger.warn(`Failed to create resource ${resourceId.path}: ${createErrorMessage(err)}`);
+      }
+    }
```

Comment:
So a solution that might work is that in `TemplatedResourcesGenerator`, the method `generateResource` checks whether a given container already exists.
When it does, no yield will happen and otherwise we continue as it is.

Reply:
That is what I was thinking of so would be great if that worked.

Reply:
It does work!